### PR TITLE
Use Node 18.15

### PR DESCRIPTION
**Pull request by alexweininger**
VS Code has updated to Node 18.15: https://code.visualstudio.com/updates/v1_82#_electron-25-update

### CODE REVIEW

**Comment by bwateratmsft** [comment]
We should raise the minimum VSCode version in package.json as well

File: .azure-pipelines/common/build.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 steps:
 - task: NodeTool@0
-  displayName: 'Use Node 16.x'
+  displayName: 'Use Node 18.x'
   inputs:
-    versionSpec: 16.x
+    versionSpec: 18.x
```
Comment (bwateratmsft) [inline .azure-pipelines/common/build.yml:5]: An alternative that puts version into exactly one place (.nvmrc):
steps:
```yaml
  - task: NodeTool@0
    displayName: Using Node.js
    inputs:
      versionSource: fromFile
      versionFilePath: .nvmrc
```

Comment (alexweininger) [reply .azure-pipelines/common/build.yml:5]: Oh nice. Is that new? I thought I looked for this option when I added the .nvmrc files

Comment (bwateratmsft) [reply .azure-pipelines/common/build.yml]: I'm not sure

File: package.json [hunk 2]
```diff
@@ -1150,7 +1150,7 @@
         "@types/node": "^16.18.36",
         "@types/ps-tree": "^1.1.0",
         "@types/semver": "^5.5.0",
-        "@types/vscode": "1.79.0",
+        "@types/vscode": "^1.81.0",
```
Comment (alexweininger) [inline package.json:1153]: This is the latest types version. Not sure why 1.82.0 doesn't exist.

Comment (bwateratmsft) [reply package.json:1153]: I've seen that before where the `@types/vscode` package lags behind VSCode itself

**Review by bwateratmsft — APPROVED** [review]

**Pull request merged**